src/app.js

```diff
@@ -0,0 +1,28 @@
+var allnews = new ArticleList()
+
+fetch("http://content.guardianapis.com/search?show-fields=all&q=uk&page-size=12&api-key=fc20c2bc-137d-4f7b-a6c3-3c4efb09563d")
```

Comment:
Ooh, I didn't know 'fetch' was a thing! Looks like it's only been around for a few months, I just checked the API docs (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#Specifications) and it looks like it's not supported by IE (something to beware of with newer Javascript features) but full marks for using it and also for making use of promises.